gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdksExtension.java

```diff
@@ -54,8 +57,19 @@ public JdksExtension() {
                 MapProperty.class, getObjectFactory().mapProperty(String.class, String.class));
         this.jdkStorageLocation = SynchronizedInterface.synchronizeAllInterfaceMethods(
                 DirectoryProperty.class, getObjectFactory().directoryProperty());
+        this.daemonTarget = project.getObjects().property(JavaLanguageVersion.class);
+
         this.getCaCerts().finalizeValueOnRead();
         this.getJdkStorageLocation().finalizeValueOnRead();
+        daemonTarget.finalizeValueOnRead();
+    }
+
+    public final Property<JavaLanguageVersion> getDaemonTarget() {
+        return daemonTarget;
+    }
+
+    public final void setDaemonTarget(String value) {
+        daemonTarget.set(ChosenJavaVersion.fromString(value).javaLanguageVersion());
```

Comment:
I'd avoid using baseline's `ChosenJavaVersion` here (we don't need the preview functionality) - might as well just use `JavaLanguageVersion.of`.